docs/configuration.md

```diff
@@ -38,84 +38,87 @@ The prevelance for all Options is as follows:
+| `exchange.markets_refresh_interval` | The interval in minutes in which markets are reloaded. <br>*Defaults to `60` minutes.*
+| `edge.*` | Please refer to [edge configuration document](edge.md) for detailed explanation.
+| `experimental.block_bad_exchanges` | Block exchanges known to not work with freqtrade. Leave on default unless you want to test if that exchange works now. <br>*Defaults to `true`.* 
+| `pairlists` | Define one or more pairlists to be used. [More information below](#dynamic-pairlists). <br>*Defaults to `StaticPairList`.* 
+| `telegram.enabled` | Enable the usage of Telegram.
+| `telegram.token` | Your Telegram bot token. Only required if `telegram.enabled` is `true`. **Keep it in secrete, do not disclose publicly.**
+| `telegram.chat_id` | Your personal Telegram account id. Only required if `telegram.enabled` is `true`. **Keep it in secrete, do not disclose publicly.**
+| `webhook.enabled` | Enable usage of Webhook notifications
+| `webhook.url` | URL for the webhook. Only required if `webhook.enabled` is `true`. See the [webhook documentation](webhook-config.md) for more details.
+| `webhook.webhookbuy` | Payload to send on buy. Only required if `webhook.enabled` is `true`. See the [webhook documentationV](webhook-config.md) for more details.
+| `webhook.webhooksell` | Payload to send on sell. Only required if `webhook.enabled` is `true`. See the [webhook documentationV](webhook-config.md) for more details.
+| `webhook.webhookstatus` | Payload to send on status calls. Only required if `webhook.enabled` is `true`. See the [webhook documentationV](webhook-config.md) for more details.
+| `db_url` | Declares database URL to use. NOTE: This defaults to `sqlite://` if `dry_run` is `True`, and to `sqlite:///tradesv3.sqlite` for production instances.
+| `initial_state` | Defines the initial application state. More information below. <br>*Defaults to `stopped`.*
+| `forcebuy_enable` | Enables the RPC Commands to force a buy. More information below. 
+| `strategy` | **Required** Defines Strategy class to use. Recommended to set via `--strategy NAME`.
+| `strategy_path` | Adds an additional strategy lookup path (must be a directory).
+| `internals.process_throttle_secs` | Set the process throttle. Value in second. <br>*Defaults to `5` seconds.*
+| `internals.heartbeat_interval` | Print heartbeat message every X seconds. Set to 0 to disable heartbeat messages. <br>*Defaults to `60` seconds.*
+| `internals.sd_notify` | Enables use of the sd_notify protocol to tell systemd service manager about changes in the bot state and issue keep-alive pings. See [here](installation.md#7-optional-configure-freqtrade-as-a-systemd-service) for more details.
+| `logfile` | Specify Logfile. Uses a rolling strategy of 10 files, with 1Mb per file.
```

Comment:
"Specifies logfile name. Uses a rolling strategy for log file rotation for 10 files with the 1MB limit per file."